lexic.el

```diff
@@ -662,21 +662,33 @@ Returns a list of plists with keys :word, :dict, and :info."
                     (list (append (cadr (assoc dict suggestions)) (list word))))
           (setq suggestions (append suggestions `((,dict . ((,word)))))))))
     (concat
+     "\u200B"
      (propertize
       (replace-regexp-in-string
        "items" "entries"
        (substring results 0 (string-match "\n" results)))
       'face 'warning)
      "\n"
      (mapconcat (lambda (dict-suggestions)
-                  (format "\u200B\u200B%s\n\u200B\u200B\u200B%s"
+                  (format "%s\n%s"
                           (propertize (or
                                        (lexic-dictionary-spec (car dict-suggestions) :short)
                                        (car dict-suggestions))
                                       'face 'outline-3)
-                          (propertize
-                           (mapconcat #'identity (cadr dict-suggestions) "\n\u200B\u200B\u200B")
-                           'face 'font-lock-keyword-face)))
+			  (mapconcat (lambda (entry)
+				       (propertize
+					(format "\u200B\u200B%s" entry)
+					;; (mapconcat #'identity (cadr dict-suggestions) "\n\u200B\u200B\u200B")
+					'face 'font-lock-keyword-face
+					'keymap (let ((map (make-sparse-keymap)))
+						  (define-key map (kbd "<RET>")
+							      (lambda ()
+								(interactive)
+								(lexic-search entry nil (list (car dict-suggestions)) t)
+								(setq lexic-current-dictionary-list t)))
```

Comment:
A hack(?) to reset the lexic-current-dictionary-list. Without this, lexic remembers the particular dictionary until called with prefix argument.
Not specifying the dict-list argument of lexic-search results in the "suggestions" buffer again.